### ISSUE TITLE
Means of getting neural network logits

### ISSUE THREAD

**Issue by xd009642** [emotechlab/silero-rs#15]
Plotting out the neural network logits for each test chunk would be very useful for debugging and may be desired by a mysterious future user who doesn't want our logic/processing that exists around the vad.

It might be nice for this to be it's own method that the current `process_internal` calls.
